src/formatDate.js

```diff
@@ -49,8 +49,53 @@
  * @returns {string}
  */
 
+// "use strict";
+
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const toFormatDate = [];
+  const dateArray = date.split(fromFormat[fromFormat.length - 1]);
+  const shortYYFormat = toFormat.indexOf('YY');
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    for (let j = 0; j < toFormat.length - 1; j++) {
+      if (fromFormat[i] === toFormat[j]) {
+        toFormatDate[j] = dateArray[i];
+      }
+
+      if (
+        fromFormat.includes('YYYY')
+        && toFormat.includes('YY')
```

Comment:
`fromFormat.includes('YYYY') && toFormat.includes('YY')` this condition is duplicated, so make sense to create separate if for that